README.md

```diff
@@ -427,4 +427,28 @@ Example usage
         github-token: ${{ steps.get-token.outputs.token }}
         threshold: ${{ inputs.inclint-threshold }}
 
-```
\ No newline at end of file
+```
+
+## JavaScript
+
+### Lint
+
+This action runs [prettier](https://prettier.io) and [eslint](https://eslint.org/)
+on the target repository.
+This action does not provide any configuration for the linters,
+it relies on a configfile (such as `package.json`) located in the target repository.
+
+```yaml
+# Basic usage
+      - uses: actions/checkout@v3
```

Comment:
pin action to `ac593985615ec2ede58e132d2e21d2b1cbd6127c` instead of `v3`

Reply:
This is just an example in Readme. Also this is a built-in github action and I didn't see it pinned in any other our repository

Reply:
```suggestion
      - uses: actions/checkout@SHA_COMMIT # check https://piwikpro.atlassian.net/wiki/spaces/EN/pages/85979955404/Github+actions+integration+guide#External-actions-must-be-pinned-to-specific-commits
```

Reply:
So if it's readme maybe this will be more adequate ?

Reply:
The same issue happens 8 times in this same file. I think it should be addressed globally by the security team in a separate PR.